app/src/main/java/com/github/bgabriel998/softwaredevproject/CameraUiView.java

```diff
@@ -28,24 +26,30 @@
     private Paint secondaryLinePaint;
     private Paint terciaryLinePaint;
     private Paint mainTextPaint;
+    private Paint mountainInfo;
     private Paint secondaryTextPaint;
 
     //Colors of the compass-view
     private int compassColor;
+    private int mountainInfoColor;
 
     //Font size of the text
     private int mainTextSize;
 
     //Max opacity for the paints
     private static final int MAX_ALPHA = 255;
 
+    //Rotation to be applied for the addition info of the mountains
+    private static final int LABEL_ROTATION = -45;
+    //Offset for the text to be above the marker
+    private static final float OFFSET_MOUNTAIN_INFO = 15;
```

Comment:
Maybe as it's a final attribute we can put it as an int, as it is not assigned a float number.
```suggestion
    private static final int OFFSET_MOUNTAIN_INFO = 15;
```